docs/tileentities/tileentity.md

```diff
@@ -0,0 +1,128 @@
+# TileEntities
+
+Tile Entities are like simplified Entities, that are bound to a Block.
+They are used to store dynamic data, execute tick based tasks and for dynamic rendering.
+Some examples would be, vanilla Minecraft to handle inventories on chests, smelting logic on furnaces, or area effects for beacons. 
+More advanced examples exist in mods, such as quarries, sorting machines, pipes, and displays.
+
+!!! note
+    `TileEntities` aren't a solution for everything and they can cause lag when used in large amounts.
```

Comment:
This is misleading information. Specifically *ticking* TEs (they don't tick by default) have the potential for lag. It might be better to mention this pitfall in the ITickable section.